cluster/mesos/docker/mesos-slave/bin/wrapdocker

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+# Copyright 2013-2014 Jérôme Petazzoni, Johan Haleby, lalyos, James Harris,
```

Comment:
The original license is Apache 2 as well. Compare https://github.com/jpetazzo/dind/blob/master/wrapdocker and the License file in the same repo.